packages/@react-spectrum/s2/src/Accordion.tsx

```diff
@@ -0,0 +1,368 @@
+
+
+export interface AccordionGroupProps extends UnsafeStyles, DOMProps, SlotProps {
+  /** The accordion items in the accordion group. */
+  children: React.ReactNode,
+  /** Spectrum-defined styles, returned by the `style()` macro. */
+  styles?: StyleString,
+  /**
+   * The size of the accordion.
+   * @default "M"
+   */
+  size?: 'S' | 'M' | 'L' | 'XL',
+  /**
+   * The amount of space between the accordion items.
+   * @default "regular"
+   */
+  density?: 'compact' | 'regular' | 'spacious',
+  /** Whether the accordion should be displayed with a quiet style. */
+  isQuiet?: boolean,
+  /** Whether the accordion should be disabled. */
+  isDisabled?: boolean
+}
+
+const groupStyles = style({
+  display: 'flex',
+  flexDirection: 'column',
+  minWidth: sizeValue(200)
+});
+
+export const AccordionGroupContext = createContext<ContextValue<AccordionGroupProps, DOMRefValue<HTMLDivElement>>>(null);
+
+let InternalAccordionGroupContext = createContext<Pick<AccordionGroupProps, 'size'| 'isQuiet'| 'density' | 'isDisabled'>>({});
+
+function AccordionGroup(props: AccordionGroupProps, ref: DOMRef<HTMLDivElement>) {
+  [props, ref] = useSpectrumContextProps(props, ref, AccordionGroupContext);
+  let domRef = useDOMRef(ref);
+  let {
+    UNSAFE_style,
+    UNSAFE_className = '',
+    size = 'M',
+    density = 'regular',
+    isQuiet,
+    isDisabled,
+    ...otherProps
+  } = props;
+  const domProps = filterDOMProps(otherProps);
+  return (
+    <Provider
+      values={[
+        [InternalAccordionGroupContext, {size, isQuiet, density, isDisabled}]
+      ]}>
+      <div
+        {...domProps}
+        ref={domRef}
+        style={UNSAFE_style}
+        className={(UNSAFE_className ?? '') + mergeStyles(groupStyles, props.styles)}>
+        {props.children}
+      </div>
+    </Provider>
+  );
+}
+
+/**
+ * A accordion group is a container for multiple accordion items.
+ */
+let _AccordionGroup = /*#__PURE__*/ (forwardRef as forwardRefType)(AccordionGroup);
+export {_AccordionGroup as AccordionGroup};
+
+const itemStyles = style({
+  color: 'heading',
+  padding: '[2px]'
+});
+
+export interface AccordionItemProps extends RACAccordionItemProps, UnsafeStyles, DOMProps {
+  children: React.ReactNode
+}
+
+export const AccordionItemContext = createContext<ContextValue<AccordionItemProps, DOMRefValue<HTMLDivElement>>>(null);
+
+const lastDividerStyles = style({
+  display: {
+    default: 'none',
+    ':last-child': 'block'
+  }
+});
+
+function AccordionItem(props: AccordionItemProps, ref: DOMRef<HTMLDivElement>) {
+  [props, ref] = useSpectrumContextProps(props, ref, AccordionItemContext);
+  let domRef = useDOMRef(ref);
+  let {
+    UNSAFE_style,
+    UNSAFE_className = '',
+    isDisabled,
+    ...otherProps
+  } = props;
+  const domProps = filterDOMProps(otherProps);
+  let {isQuiet, isDisabled: isGroupDisabled} = useContext(InternalAccordionGroupContext);
+  return (
+    <>
+      {!isQuiet && <Divider size="S" />}
```

Comment:
I don't think these are divider, i think they are just borders. In the Figma files they don't have rounded caps.
I think we don't want them as dividers in the accessibility tree either